proto.rs/src/lib.rs

```diff
@@ -10,6 +10,12 @@ mod chariott {
     pub mod common {
         pub mod v1 {
             tonic::include_proto!("chariott.common.v1");
+            pub use fulfillment::Fulfillment as FulfillmentEnum;
```

Comment:
nit: Worth adding a comment why this is done so?

Reply:
Done in f56af3343b09e05ccf13fe9ae3e483ece5f4e851